rajatuiwebdev/badge-website
# Added js directory

## What does this PR do?

Added a directory named js to handle the javascript code seperately instead of writing it in directly in index.html file. This will make the code more readable and manageable. No changes done to the code inside the scripts, made 3 different javascript files as per the scripts written in index.html file

## Type of change



- Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- We can run the website directly to check if all functionalities are working fine.

## Mandatory Tasks
